### PR TITLE
0017839: Fix issue where controller_public_ip no longer being outputted after successful apply

### PR DESCRIPTION
**Pull request by zhiqiangjin**

### CODE REVIEW

File: aviatrix-controller-build/outputs.tf [hunk 1]
```diff
@@ -1,10 +1,10 @@
 output private_ip {
   value       = length(aws_instance.aviatrixcontroller) > 0 ? aws_instance.aviatrixcontroller.*.private_ip[0] : ""
-  description = "Private IPs for first aviatrix conroller"
+  description = "Private IP for the first aviatrix conroller"
 }
 
 output public_ip {
-  value       = length(aws_instance.aviatrixcontroller) > 0 ? aws_instance.aviatrixcontroller.*.public_ip[0] : ""
+  value       = length(aws_eip.controller_eip) > 0 ? aws_eip.controller_eip.*.public_ip[0] : ""
```
Comment (arvindsree) [inline aviatrix-controller-build/outputs.tf:7]: in the case where EIP gets detached, the original code would display the IP accurately

Comment (zhiqiangjin) [reply aviatrix-controller-build/outputs.tf:7]: The original code does not output public_ip.

File: aviatrix-controller-build/outputs.tf [hunk 2]
```diff
@@ -26,3 +26,13 @@ output security_group_id {
 output name_prefix {
   value = var.name_prefix
 }
+
+output list_private_ip {
```
Comment (arvindsree) [inline aviatrix-controller-build/outputs.tf:30]: rename to private_ip_list

File: aviatrix-controller-build/outputs.tf [hunk 3]
```diff
@@ -26,3 +26,13 @@ output security_group_id {
...
+  value       = aws_instance.aviatrixcontroller.*.private_ip
+  description = "List of Private IPs for all of the aviatrix conroller launched"
+}
+
+output list_public_ip {
```
Comment (arvindsree) [inline aviatrix-controller-build/outputs.tf:35]: renamed to public_ip_list

File: aviatrix-controller-build/outputs.tf (hunk 2 above)
Comment (arvindsree) [reply aviatrix-controller-build/outputs.tf:30]: Update the two variables in README.md as well

**Pull request merged**